src/components/organisms/ChatSidebar/ChatSidebar.scss

```diff
@@ -18,6 +18,7 @@ $header-height: 45px;
   &--expanded {
     transform: translateX(0);
     transition: transform 0.3s;
+    opacity: 90%;
```

Comment:
I believe `opacity` will set the transparency on the entire div and it's contents won't it? Do we want to just be setting the transparency on the background itself here?|

(cc // @mcflyok @mobilevinay @sparkletown/product @sparkletown/engineering)

Further Reading:

- https://www.w3schools.com/css/css_image_transparency.asp
  - > Transparent Box: When using the opacity property to add transparency to the background of an element, all of its child elements inherit the same transparency. This can make the text inside a fully transparent element hard to read
  - > Transparency using RGBA: If you do not want to apply opacity to child elements, like in our example above, use RGBA color values. The following example sets the opacity for the background color and not the text:

Reply:
Looking at @margulies fork/branch, this appears to be how he set the sidebar transparent:

- https://github.com/margulies/sparkle/blob/primedre/src/components/organisms/ChatSidebar/ChatSidebar.scss#L15-L16

```scss
//background-color: $saturated-black;
  background: $saturated-black-gradient;
```

Reply:
Just to chime in here, I implemented the opacity gradient for the chat sidebar because setting a single background alpha value didn't seem to balance the need for a clear edge delineating the left side of the sidebar with the need to show a hint of the map content. This effect might be improved well with a shadow as well...